packages/@pollyjs/adapter-node-http/src/-private/transport-wrapper.js

```diff
@@ -138,15 +138,12 @@ export default class TransportWrapper {
       ...NodeUrl.parse(pollyRequest.url)
     });
 
-    request.on('error', error => {
```

Comment:
Whats the reason for removing this? By placing this here, we can throw on errors emitted during writing chunks to the request.

Reply:
The error subscription was moved to line 146 so that it could reject the promise instead of throwing an unhandled error.  I'll try adding the subscription before calling `request.write` so we don't miss out on errors writing the request body.

Reply:
Im not entirely sure what the difference in outcome is with the change. Both will result in the async method's promise to be rejected which you've caught on line 280.

Reply:
The key difference appears to be throwing an error inside the callback function. This causes the `uncaughtException` event to fire because the thrown error is not attributed to the current async state in `passthrough`, as the error callback is invoked asynchronously when the 'error' event is fired.

I've made a simple express instance that repro's the behavior before and after the proposed change here: 
https://glitch.com/edit/#!/voracious-pilot

Reply:
AHHH! Ok I see it now! Thanks for the clarification.